tests/selenium/conftest.py

```diff
@@ -91,6 +91,11 @@ def get_text(self):
             else el.get_attribute("innerHTML")
         )
 
+    def move_to(self):
+        ac = ActionChains(self.mixin.driver)
+        ac.move_to_element(self._get_cell_value())
+        return ac.perform()
+
```

Comment:
Additional table cell helper moving the mouse over it - and scrolling as required to make it visible. Has the nice side-effect of triggering the tooltip if there's one.